.travis.yml

```diff
@@ -100,13 +100,11 @@ matrix:
     # Old Version GCC 4.4
     - name: "Linux GCC 4.4 Build"
       os: linux
-      addons:
-        apt:
-          sources:
-            - ubuntu-toolchain-r-test
-          packages:
-            - gcc-4.4
-            - gdb
+      before_install:
```

Comment:
This is the only line change I did for PIC:

sources:
    - sourceline: 'ppa:ubuntu-toolchain-r/test'